### PR TITLE
More robust telephone

### PR DESCRIPTION
**Pull request by raoulsullivan**
Bugs:
* Deals with data completion errors when translating objects from sme_contact_v2 and v3 to person_v1 and address_v1

### CODE REVIEW

File: sme_finance_application_schema/translations.py [hunk 1]
```diff
@@ -251,8 +251,8 @@ def sme_v5_to_aggregated_actors_v1_translator(sme):
 
 def sme_contact_v3_to_address_v1_translator(sme_contact):
     address = {
-        'building_number_and_street_name': sme_contact.get('address_line_1') or '',
-        'postcode': sme_contact.get('postcode') or '',
+        'building_number_and_street_name': sme_contact.get('address_line_1') or 'Unknown',
```
Comment (jmstaley) [inline sme_finance_application_schema/translations.py]: any reason not to use `sme_contact.get('address_line_1', 'Unknown')` get can return a default

Comment (raoulsullivan) [reply sme_finance_application_schema/translations.py]: No, no reason at all. The `.get` syntax is also easier on the eye

Comment (jmstaley) [reply sme_finance_application_schema/translations.py]: acutally, talking with james, if you use get it won't default to 'Unknown' on an empty string using gets default

Comment (raoulsullivan) [reply sme_finance_application_schema/translations.py]: Ah, yes there is. This is one Anthony generally protected us against

```
>>> address = {'postcode': ''}
>>> address.get('postcode', 'Unknown')
''
>>> address.get('postcode') or 'Unknown'
'Unknown'
```

**Pull request merged**